engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1830,14 +1831,15 @@ static void LogFrameBufferError(GLenum status)
         glGetProgramiv(p, GL_LINK_STATUS, &status);
         if (status == 0)
         {
+            dmLogError("Unable to link program.");
 #ifndef NDEBUG
             GLint logLength;
             glGetProgramiv(p, GL_INFO_LOG_LENGTH, &logLength);
             if (logLength > 0)
             {
                 GLchar *log = (GLchar *)malloc(logLength);
                 glGetProgramInfoLog(p, logLength, &logLength, log);
-                dmLogWarning("%s\n", log);
+                dmLogError("%s", log);
```

Comment:
Usually after this error, the game on Defold crashes as it's a significant error. Plus, Warning won't get into the error reporting system as done in the Poki SDK. So I replaced it with dmLogError.